docs/platforms/python/performance/index.mdx

```diff
@@ -18,7 +18,7 @@ If you’re adopting Performance in a high-throughput environment, we recommend
 
 <PlatformContent includePath="performance/configure-sample-rate" />
 
-Learn more about performance monitoring <PlatformLink to="/configuration/options/#tracing-options">options</PlatformLink>, or how to <PlatformLink to="/configuration/sampling/#sampling-transaction-events">sample transactions</PlatformLink>.
+Learn more about performance monitoring <PlatformLink to="/configuration/options/#tracing-options">options</PlatformLink>, how to use the <PlatformLink to="/configuration/sampling/#setting-a-sampling-function">tracesSampler</PlatformLink> function, or how to <PlatformLink to="/configuration/sampling/#sampling-transaction-events">sample transactions</PlatformLink>.
```

Comment:
```suggestion
Learn more about performance monitoring <PlatformLink to="/configuration/options/#tracing-options">options</PlatformLink>, how to use the <PlatformLink to="/configuration/sampling/#setting-a-sampling-function">traces_sampler</PlatformLink> function, or how to <PlatformLink to="/configuration/sampling/#sampling-transaction-events">sample transactions</PlatformLink>.
```
Hmm, looks like some platforms call it traces_sampler as opposed to tracesSampler. There might be some other variations as well. Maybe we should just call it "how to use the sampling function" instead of "how to use the tracesSampler" throughout?